CONTRIBUTING.md

```diff
@@ -188,8 +188,9 @@ to merge the pull request and `pub submit` any affected packages.
 You must complete the
 [Contributor License Agreement](https://cla.developers.google.com/clas).
 You can do this online, and it only takes a minute.
-If you've never submitted code before, you must add your (or your
-organization's) name and contact info to the [AUTHORS](AUTHORS) file.
+If you've never submitted code for that plugin before, you may also add your (or
+your organization's) name and contact info to the AUTHORS file for the plugin.
+You may also add it to the AUTHORS file for [the repository](AUTHORS).
```

Comment:
Should we compute the initial version of the per-plugin AUTHORS from the git log?

Reply:
I considered it, but it would be significantly more time-intensive, and I couldn't see any real benefit. The authors file isn't a legal source of truth (the CLA and the git history are the definitive information, as I understand it), just a way to recognize contributions, and I don't think anyone will be unhappy about being slightly over-represented.

It's an odd situation in the first place since normally there is just one such file in a repo; parsing contributions by subfolder isn't the norm. Looking at it another way: if someone currently wanted to find out who "The Flutter Authors" referred to in a specific plugin, they would find exactly this same list, it would just be harder to do so, so nothing is fundamentally changing.

Reply:
Do we want a per-package AUTHORS, then? flutter/flutter has a root AUTHORS file, but then each individual package doesn't have an individual one.

Reply:
Hixie wanted these because the published packages (i.e., what is downloaded from pub.dev) have a license file that reference "the authors", but no authors list, which is kind of weird. (This is weird in a different way, but here the weirdness is restricted to the repo instead of what we distribute.)

I assume nothing in flutter/flutter is published as a package.